src/coreclr/vm/stackwalk.cpp

```diff
@@ -716,11 +716,35 @@ UINT_PTR Thread::VirtualUnwindToFirstManagedCallFrame(T_CONTEXT* pContext)
     CONTRACTL_END;
```

Comment:
@janvorli, for a long term; can we encode some sort of a sentinel in the header of first managed frame (or leaf managed frame at the managed->native boundary) so we can walk up to it for searching? That will avoid one more libunwind/VirtualUnwind call (we have six remaining calls before we can completely get rid of that dependency 🙂).